server/handlers/changefeed.go

```diff
@@ -63,7 +63,7 @@ func checkChangefeedInputs(logger ctxu.Logger, s interface{}, r string) (
 
 	store, ok := s.(storage.MetaStore)
 	if !ok {
-		logger.Error("500 GET unable to retrieve storage")
+		logger.Errorf("%d GET unable to retrieve storage", http.StatusInternalServerError)
 		err = errors.ErrNoStorage.WithDetail(nil)
 		return
 	}
```

Comment:
nit: could you also update the `logger.Errorf("400...` below to use `http. StatusBadRequest`?